decidim-debates/spec/system/user_edits_debate_spec.rb

```diff
@@ -49,6 +50,55 @@
       expect(page).to have_css("[data-author]", text: user.name)
     end
 
+    context "when attachments are disallowed" do
+      it "does not show the attachments form" do
+        visit_component
+
+        click_on debate.title.values.first
+        find("#dropdown-trigger-resource-#{debate.id}").click
+        click_on "Edit"
+
+        expect(page).to have_no_css("#debate_documents_button")
+      end
+    end
+
+    context "when attachments are allowed", :slow do
```

Comment:
Could you add a spec with what happens what if you upload a document that is not allowed? for example a ODT file ?